### ISSUE TITLE
Feature Request: Option to disable searching Obsidian note titles

### ISSUE THREAD

**Issue by hungsu** [PKM-er/Obsidian-Surfing#261]
### Feature Requested

I have a vault with over 10000 files. When typing into the Surfing address bar, it freezes because it's trying to search my note titles in addition to addresses. Could we have an option to not search local files at all from the Surfing address/search bar?

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by CasanovaSan** [comment]
YES THIS IS ABSOLUTELY NEEDED!!! the only way i can use the extension is either by pulling the vault out or using links and immediately pressing enter
